docs/workspaces.md

```diff
@@ -364,14 +364,98 @@ write to or read from that `Workspace`. Use the `runAfter` field in your `Pipeli
 to define when a `Task` should be executed. For more information, see the [`runAfter` documentation](pipelines.md#using-the-runafter-parameter).
 
 When a `PersistentVolumeClaim` is used as volume source for a `Workspace` in a `PipelineRun`,
-an Affinity Assistant will be created. The Affinity Assistant acts as a placeholder for `TaskRun` pods
+an Affinity Assistant will be created. Available Affinity Assistant Modes are **coschedule workspace**, **coschedule pipelineruns**,
+**isolate pipelineruns** and **disabled**. 
+
+* **coschedule workspace** - The Affinity Assistant acts as a placeholder for `TaskRun` pods
```

Comment:
worth noting that with this mode, you can only mount 1 pvc backed workspace to each taskrun

Reply:
SGTM, added in the new file